docs/qiita-create-study.rst

```diff
@@ -0,0 +1,164 @@
+   :align:   center
+
+.. figure::  images/image07.png
+   :align:   center
+
+The "New User" link brings you to a page on which you can create a new
+account. Optional fields are indicated explicitly, while all other
+fields are required. Once the form is submitted, an email will be sent
+to you containing instructions on how to verify your email address.
+
+Once you have created your account, you can log into the system by
+entering your email and password.
+
+.. figure::  images/image03.png
+   :align:   center
+
+
+Creating a test study
+---------------------
+
+Studies are the main source of data for Qiita. Studies can contain only one set
+of samples but can contain multiple sets of raw data, each of which can have a
+different preparation -- for example, 16S, shotgun metagenomics, and metabolomics.
+
+In this tutorial, our study contains 12 samples, each with two types of data:
+16S and shotgun metagenomics. To represent this project in Qiita, you will need
+to create a single study with a single sample information file that contains all
+12 samples. Then, you will link separate preparation files for each data type.
+
+Creating an example study
+-------------------------
+
+To create a study, click on the "Study" menu and then on "Create Study".
+This will take you to a new page that will gather some basic information
+to create your study.
+
+.. figure::  images/image18.png
+   :align:   center
+
+The "Study Title" has to be unique system-wide. Qiita will check this
+when you try to create the study, and may ask you to alter the study
+name if the one you provide is already in use.
+
+.. figure::  images/image02.png
+   :align:   center
+
+A principal investigator is required, and a list of known PIs is
+provided. If you cannot find the name you are looking for in this
+list, you can choose to add a new one.
+
+Select the environmental package appropriate to your study. Different
+packages will request different specific information about your samples.
+This information is optional; for more details, see the metadata
+section.
+
+There is also an option to specify time series type ("Event-Based Data") if you
+have time series data. In our case, the samples come from a cross-sectional
+study design, so you should select "No time series." For more information on
+time series types, you can check out the
+`in-depth tutorial <https://qiita.ucsd.edu/static/doc/html/tutorials/getting-started.html#creating-a-study>`__
+on the Qiita website.
+
+Once your study has been created, you will be informed by a green
+message; click on the study name to begin adding your data.
+
+.. figure::  images/image04.png
+   :align:   center
+
+
+Adding sample information
+-------------------------
+
+**Sample information** is the set of metadata that pertains to your biological
+samples: there are the measured variables that are motivating you to look for
+response variables in the microbiome. (To learn more, including how to format
```

Comment:
microbiome. **IMPORTANT:** your metadata are your study; it is imperative that those data are consistent, correct, and sufficiently detailed.